src/Agent.Sdk/Util/ParentProcessUtil/WindowsParentProcessUtil.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Agent.Sdk.Util.ParentProcessUtil
+{
+    public enum ParentProcessNames { Pwsh, Powershell };
```

Comment:
I thought the issue was only running under ps core (pwsh), so no need to warn if powershell?

Reply:
It was implemented this way initially before my changes. Fixed it